capacity4more/modules/c4m/features/c4m_features_homepage/templates/voices-views-carousel.tpl.php

```diff
@@ -5,14 +5,16 @@
     <ul rn-carousel rn-carousel-index="carouselIndex" rn-carousel-loop rn-carousel-controls class="carousel">
       <li ng-repeat="slide in carouselImages track by slide.id" ng-class="'id-' + slide.id">
         <div ng-style="{'background-image': 'url(' + slide.image + ')'}"  class="bgimage">
-          <div class="intro-text">
-            <span class="title">
-            {{ slide.title }}
-          </span>
-          <span class="text">
-            {{ slide.text }}
-          </span>
-          </div>
+          <a href="/articles">
```

Comment:
No, use ``<?php print url('articles') ?>`` Disable pURL if you have to by sending the options to the template from the plugin.